core-api/src/main/java/com/optimizely/ab/internal/EventTagUtils.java

```diff
@@ -47,4 +47,22 @@ public static Long getRevenueValue(@Nonnull Map<String, ?> eventTags) {
         }
         return eventValue;
     }
+
+    /**
+     * Fetch the numeric metric value from event tags. "value" is a reserved keyword.
+     */
+    public static Double getNumericValue(@Nonnull Map<String, ?> eventTags) {
+        Double eventValue = null;
+        if (eventTags.containsKey(ReservedEventKey.VALUE.toString())) {
+            Object rawValue = eventTags.get(ReservedEventKey.VALUE.toString());
+            if (rawValue instanceof Double) {
```

Comment:
makes sense, the opposite case is the one that we should be wary of (truncating floating-point values)